README.adoc

```diff
@@ -44,10 +44,10 @@ There is many other ways one can help us: packaging, communication, etc ...
  * https://github.com/apache/james-project/#how-to-release-via-maven-release-plugin[How to release via maven release plugin]
  * https://github.com/apache/james-project/#how-to-check-the-compilation[How to check the compilation]
  * https://github.com/apache/james-project/#how-to-run-james-in-docker[How to run James in Docker]
- ** https://github.com/apache/james-project/#run-james-with-java-8--guice--cassandra--rabbitmq--swift--elasticsearch[Run James with Java 8 + Guice + Cassandra + RabbitMQ + Swift + ElasticSearch]
- ** https://github.com/apache/james-project/#run-james-with-java-8--guice--cassandra--elasticsearch[Run James with Java 8 + Guice + Cassandra + ElasticSearch]
- ** https://github.com/apache/james-project/#run-james-with-java-8--guice--jpa--lucene[Run James with Java 8 + Guice + JPA + Lucene]
- ** https://github.com/apache/james-project/#run-james-with-java-8--spring--jpa[Run James with Java 8 + Spring + JPA]
+ ** ++https://github.com/apache/james-project/#run-james-with-java-8--guice--cassandra--rabbitmq--swift--elasticsearch++[Run James with Java 8 + Guice + Cassandra + RabbitMQ + Swift + ElasticSearch]
+ ** ++https://github.com/apache/james-project/#run-james-with-java-8--guice--cassandra--elasticsearch++[Run James with Java 8 + Guice + Cassandra + ElasticSearch]
+ ** ++https://github.com/apache/james-project/#run-james-with-java-8--guice--jpa--lucene++[Run James with Java 8 + Guice + JPA + Lucene]
+ ** ++https://github.com/apache/james-project/#run-james-with-java-8--spring--jpa[Run James with Java 8 + Spring + JPA]++
```

Comment:
that's worst, it's breaking formatting :/ please remove this last fixup, we'll fix this unrelated comment in another pr/issue